metabase/metabase
# fix: Don't show error page when command palette documentation link is clicked

Closes #47829

### Description

The bug: when you click or press enter on a command palette documentation link, the documentation opens in another tab, but the page where the command palette was displayed changes to an error page. In this branch, we fix this problem by preventDefault'ing the link's normal behavior.

### How to verify

Search for "asdf" in the command palette. You'll see a documentation link below the results. ("asdf" ensures that there are no results, which is convenient.) Try the following:

* Click the link
* Press enter while the link is selected
* Right click and open in new tab
* Middle click to open a new tab (if this works in your environment)

[Loom](https://www.loom.com/share/de524315c8e54751aaf494bbf05fd87a?sid=de1af7c8-4bf0-4d87-b5dc-20357eab5b3e) - this shows that left clicking and middle clicking on documentation and view-results links works, whether Metabase is hosted normally or on a subpath. Pressing enter also works, but I forgot to include that in the video.
